spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -513,13 +514,11 @@ public static class Tomcat {
 		 */
 		private DataSize maxHttpResponseHeaderSize = DataSize.ofKilobytes(8);
 
-		public DataSize getMaxHttpFormPostSize() {
-			return this.maxHttpFormPostSize;
-		}
-
-		public void setMaxHttpFormPostSize(DataSize maxHttpFormPostSize) {
-			this.maxHttpFormPostSize = maxHttpFormPostSize;
-		}
+		/**
+		 * Maximum number of parameters (GET plus POST) that will be automatically parsed
+		 * by the container. A value of less than 0 means no limit.
+		 */
+		private int maxParameterCount = 10000;
```

Comment:
I’m curious—could you explain where and how this is used for conditional checks? I’d love to learn more about it.

Reply:
What do you mean by "conditional checks"? You can review the other changes in this PR to see how this property's value will be used.

Reply:
That's not our code, Tomcat would use the value set. The `factory.addConnectorCustomizers((connector) -> connector.setMaxParameterCount(maxParameterCount));` line passes the value along.